package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "eslint-config-springworks",
-  "version": "2.6.0",
+  "version": "2.7.0",
```

Comment:
- [ ] please don't change version number in a PR. That's changed when a new version is published.